docs/explanation/lifecycle-layer.rst

```diff
@@ -0,0 +1,198 @@
+
+From lifecycle to OCI layer
```

Comment:
rendered version: https://canonical-rockcraft--445.com.readthedocs.build/en/445/explanation/lifecycle-layer/

Reply:
But I think that's the problem and that's why we're discussing the `usrmerge` in the other thread below. I.e. this page addresses multiple aspects of the OCI layering process in Rockcraft. In reality, if someone reads this page, they will "Learn how Rockcraft builds an OCI layer from 'parts'".

Not trying to be a buzzkill, but I'd actually consider another approach: _we should have an explanation heading for how the rocks' layers come to be_. In that case, this page would feel natural, to me, as a subpage under the "OCI Layering" explanation heading. One more argument in favour of such a re-org is the level of knowledge that is being "explained" here. If you compare this with the other existing explanations, this is much more of a tech-dive than the others (which are more product-oriented).